### PR TITLE
Link pages to principles

### LINKED ISSUE #191: Link to the Principles page from other pages in the Guide
In #144 We've added a new page called `Principles` which you can preview here https://bitcoin.design/guide/foundations/principles/

This task is simple, whoever takes on this issue, should see where in the Guide could be a good idea to link to the `Principles page`. It can be either to the page itself or of course to a specific principle, for example, security https://bitcoin.design/guide/foundations/principles/#security

### PR DESCRIPTION
**Pull request by pavlenex**
This simple PR links to certain principles in the `Foundation` section, once the context allows it. This isn't found and replace, I made sure it's contextual and didn't really want to overdo it.

Closes #191

### CODE REVIEW

File: guide/glossary.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ BIP44 instroduced the following structure, which BIP49 and BIP84 follow:<br/>
 The path to the first address in a bitcoin-wallet using BIP84 will look like this:<br/>
 `m/84h/0h/0h/0/0`
 
-For full interoperability a wallet should support all of these standards. More information can be found [here](https://learnmeabitcoin.com/technical/derivation-paths).
+For full [interoperability](/guide/foundations/principles/#interoperability) a wallet should support all of these standards. More information can be found [here](https://learnmeabitcoin.com/technical/derivation-paths).
```
Comment (GBKS) [inline guide/glossary.md:161]: We could also link to the [Wallet interoperability page](http://localhost:4000/guide/foundations/wallet-interoperability/) in Foundations.

**Review by GBKS — APPROVED** [review]

File: guide/glossary.md (hunk 1 above)
Comment (pavlenex) [reply guide/glossary.md:161]: @GBKS  Would you prefer that I link to that page instead?

Comment (GBKS) [reply guide/glossary.md:161]: Could also add it in the more information sentence.

`More information can be found [here](https://bitcoin.design/guide/foundations/wallet-interoperability/#wallet-import-and-export) and  [here](https://learnmeabitcoin.com/technical/derivation-paths).`

Comment (pavlenex) [reply guide/glossary.md]: Done https://github.com/BitcoinDesign/Guide/pull/218#pullrequestreview-634496189

**Review by GBKS — APPROVED** [review]

**Pull request merged**